Src/VimCore/InsertUtil.fs

```diff
@@ -41,6 +41,7 @@ type internal InsertUtil
     let _vimHost = _vimBufferData.Vim.VimHost
     let _vimTextBuffer = _vimBufferData.VimTextBuffer
     let mutable _combinedEditStartPoint: ITrackingPoint option = None
+    let mutable _replaceUndoLine: ITextSnapshotLine option = None

```

Comment:
Did you consider using `ITrackingLineColumn` here? The goal here seems to be to track a line number across edits to the buffer. That is a bit better done with `ITrackingLineColumn`.

Reply:
This is literally a snapshot of the the old line before any insert edits were made on it. A tracking line column would not be able to provide us with that historical information.